trp_use_case/view/qweb_report_use_case_collection.xml

```diff
@@ -0,0 +1,139 @@
+                                    </tr>
+                                    <tr>
+                                        <td>Result</td>
+                                        <td>
+                                            <span t-field="ui.result" />
+                                        </td>
+                                    </tr>
+                                    <tr>
+                                        <td>Exception</td>
+                                        <td>
+                                            <span t-field="ui.exceptions" />
+                                        </td>
+                                    </tr>
+                                    <tr t-if="trp_use_case_internal_report">
+                                        <td>Implementation</td>
+                                        <td>
+                                            <span t-field="ui.implementation" />
+                                        </td>
+                                    </tr>
+                                    <tr>
+                                        <td>Workload</td>
+                                        <td>
+                                            <t t-foreach="ui.workload_ids" t-as="wl">
+                                                <t t-if="wl.optional">
+                                                    (Optional)
+                                                </t>
+                                                <span t-field="wl.name" />:
+                                                <span t-field="wl.hours" /> hours <br/>
+                                            </t>
+                                        </td>
+                                    </tr>
+                                </table>
+                            </t>
+                            <div class="row">
+                                <div class="col-xs-12">
+                                    Total number of hours:
+                                    <span t-field="o.hours_total" />
+                                    <t t-if="o.hours_total_optional">
+                                        of which
+                                        <span t-field="o.hours_total_optional" />
+                                        optional
+                                    </t>
+                                </div>
+                            </div>
+                        </div>
+                    </t>
+                </t>
+        </template>
+
+     <template id="qweb_report_ucc_document">
+        <t t-set="trp_use_case_internal_report" t-value="True"/>     
+        <t t-call="report.html_container">
+            <t t-foreach="doc_ids" t-as="doc_id">
+                <t t-raw="translate_doc(doc_id, doc_model, 'partner_id.lang', 'trp_use_case.qweb_report_ucc')"/>
+            </t>
+        </t>
+    </template>
+
+
+    <template id="qweb_report_ucc_document2">
```

Comment:
this one doesn't yet have a report entry in `report_use_case.xml`, I assume that's going to happen soon